pkg/helm/frrk8s.go

```diff
@@ -153,6 +155,9 @@ func frrk8sValues(envConfig params.EnvConfig, crdConfig *metallbv1beta1.MetalLB)
 		frrk8sValueMap["restartOnRotatorSecretRefresh"] = nil // the cert rotator isn't started anyways
 	}
 
```

Comment:
same, let's add a validation that the cidrs are valid to be safe?

Reply:
done